dias/chime_analyzer.py

```diff
@@ -43,3 +46,31 @@ def Finder(self, acqs=()):
         An instance of :class:`Finder`.
         """
         return finder.Finder(acqs=acqs, node_spoof={"gong": self.archive_data_dir})
+
+    def Staging_Tracker(self):
```

Comment:
This makes sense! I'm wondering if we should make this more generic though, so that you can create those just by adding to the config file.